src/services/PermissionServiceNative.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  request,
+  check,
+  requestNotifications,
+  PERMISSIONS,
+  Permission
+} from 'react-native-permissions'
+import { Platform } from 'react-native'
+import { OS_TYPES } from '../constants'
+// type used in usage examples
+type PermissionStatus =
+  | 'unavailable'
+  | 'denied'
+  | 'limited'
+  | 'granted'
+  | 'blocked'
+  | null
+
+export async function askForNotificationsPermission(): Promise<PermissionStatus> {
+  requestNotifications(['alert', 'sound']).then(({ status, settings }) => {
+    return status
+  })
+  return null
+}
+
+export async function askForPermission(
+  permissionToGet: any
+): Promise<PermissionStatus> {
+  const permissionsArray = ['CAMERA', 'PHOTO_LIBRARY']
+
+  if (permissionsArray.indexOf(permissionToGet) > 0) {
+    return null
+  }
```

Comment:
what's the purpose of this? 
seems like if you ask for `PHOTO_LIBRARY` permission just a null value will be returned and that doesn't make sense because you have some logic below regarding this...